src/App/GraphContext.ts

```diff
@@ -0,0 +1,10 @@
+import { createContext } from 'react'
+
+type GraphContextProps = {
+    graphIsOpen: boolean
+    setGraphIsOpen: (e: boolean) => void
```

Comment:
I would suggest changing the 'e' parameter to 'state' or something more explanatory